src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -14,7 +14,7 @@
 
 helps['sf application'] = """
 type: group
-short-summary: Manage applications running on an Azure Service Fabric cluster.
+short-summary: Manage applications running on an Azure Service Fabric cluster. Only supports ARM deployed applications.
```

Comment:
```suggestion
short-summary: Manage applications running on an Azure Service Fabric cluster. Only support ARM deployed applications.
```

Reply:
Align verb form with "manage".

Reply:
Thanks @qwordy, I made the change suggested